flink-kubernetes/src/test/java/org/apache/flink/kubernetes/KubernetesUtilsTest.java

```diff
@@ -144,6 +147,15 @@ public void testGetJobManagerStartCommand() {
 				" " + mainClass + " " + mainClassArgs + " " + jmLogRedirects,
 			getJobManagerStartCommand(cfg, true, true, mainClassArgs));
 
+		cfg.setInteger(ResourceManagerOptions.CONTAINERIZED_HEAP_CUTOFF_MIN, 300);
```

Comment:
I think this part seems a bit reductant, because it tries to verify the internal implementation of `BootstrapTools#calculateHeapSize` which is not brought by this PR.
IMO, this utility method should be verified completely when introduced first time. If not, we should separately verify it in general, not only in kubernetes case.

I am not strongly against if you prefer to adding it here, but another concern is that can we somehow reuse `getJobManagerExpectedCommand` in `assertEquals`?

Reply:
I will remove this part of checking cutoff. Just as you say, it has already be checked by its own tests.

We always try to reuse `getJobManagerExpectedCommand`. However, when we change the `CONTAINER_START_COMMAND_TEMPLATE` and test the behavior, we could not reuse it. They do not share lots of common codes.

Reply:
After we removing this new added part, we never have the reusing issue. And the previous ones were not in the scope of this PR. :)